src/resources/Correction/BC.ts

```diff
@@ -25,8 +25,7 @@ export const CorrectionResourceBc: ResourceIF = {
       subtitle: null // not used
     },
     nameRequestTypes: [
-      NrRequestActionCodes.CHANGE_NAME,
-      NrRequestActionCodes.CONVERSION
```

Comment:
A conversion NR is not valid for a correction (since we do not change business type in a correction).